tests/python/gaia-ui-tests/gaiatest/runtests.py

```diff
@@ -49,10 +50,27 @@ class GaiaTextTestRunner(MarionetteTextTestRunner):
     resultclass = GaiaTestResult
 
 
+class GaiaMarionetteDriver(Marionette):
```

Comment:
It was just easier than hunting down all the relevant call sites and updating them, but I can do this as you suggest instead.